parm/config/gfs/config.landensanl

```diff
@@ -0,0 +1,34 @@
+#! /usr/bin/env bash
```

Comment:
This is a duplicate of `config.landanl`.
If `landensanl` is going to use same/similar setup as `landanl`, we can just source `config.landanl` first and override the changes specific to `landensanl`.  We do this for `eobs`, `efcs` etc.

Reply:
done